ros2: documentation: fix run instructions

dlim04 opened the pull request:
- Remove link to missing [params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml)
- Complete `params.yml` file code snippet
- Replace `roslaunch` command to use `$HOME` instead of `~` path.

Fixes #361

dlim04 committed:
documentation: ros2: fix run instructions

- Remove link to missing [params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml)
- Complete `params.yml` file code snippet
- Replace `roslaunch` command to use `$HOME` instead of `~` path.

Fixes #361

robbiefish commented on README.md:
```diff
@@ -122,25 +122,29 @@ The node has some optional launch parameters that can be specified from the comm
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+- `params_file` : path to a parameter file to override the default parameters stored by default in: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
```
I would revert this. The purpose of this is to say that the defaults are stored in `params.yml`, and that should be where you look for a list of parameters

dlim04 replied:
I've realised that the reason that the `params.yml` file doesn't look like it exists is due to it being in a submodule. When you clone locally and initialise submodules the `params.yml` file appears at that location, but for GitHub markdown it  doesn't liink like that.

Looking at #337 it looks like the fix would be link to the path on the submodule's repo. Could revert and then change the path?

robbiefish replied:
Ah yeah, that caught me again. I think that reverting and then changing the path would be ideal, thanks!

robbiefish reviewed (changes requested):
Thanks for taking the time to contribute this, this is a good catch. If you revert the one line I requested, this can be merged

dlim04 committed:
documentation: ros2: add params.yml link

- Add the params.yml link to the relevant git submodule to enable GitHub
markdown linking for an example params_file.
- Add note for using the params.yml example with ROS2.

Closes #362
